tests/Settings/SettingsTest.php

```diff
@@ -157,4 +157,19 @@ public function testResetSettings(): void
         $this->assertIsArray($settings['sortableAttributes']);
         $this->assertEmpty($settings['sortableAttributes']);
     }
+
+    // Here to test for regressions related to https://github.com/meilisearch/meilisearch-php/issues/204.
```

Comment:
What do you mean "for regressions"?

Reply:
That these is the new tests to ensure that this error will not happen again